### ISSUE TITLE
False passing test for SpaceBeforeBrace linter

### ISSUE THREAD

**Issue by ackernaut** [brigade/scss-lint#177]
The second example selector block passes the SpaceBeforeBrace test. I don't think this would be a very feasible way to format, but could possibly pass unintentionally as a typo.

``` scss
// Fails
.class1{
  color: #f00;
}

// Passes
.class2
 {
  color: #f00;
}

// Passes
.class3 {
  color: #f00;
}
```
